cmd/openshift-install/main.go

```diff
@@ -40,6 +40,19 @@ func main() {
 	installerMain()
 }
 
+// Drop all terraform environment variables that the user may have passed
+// so they can't use terraform directly.
+func cleanEnvironment() {
+	environ := os.Environ()
+	for _, env := range environ {
+		splits := strings.Split(env, "=")
+		key := splits[0]
+		if strings.HasPrefix(key, "TF_") {
+			os.Unsetenv(key)
```

Comment:
Is this called early enough?  Environment mutation is [unsafe in the presence of threads](https://github.com/rust-lang/rust/pull/24741).